internal/organization/command_update.go

```diff
@@ -20,30 +20,41 @@ func (c *command) newUpdateCommand() *cobra.Command {
 	}
 
 	cmd.Flags().String("name", "", "Name of the Confluent Cloud organization.")
+	cmd.Flags().Bool("jit-enabled", false, "The flag to toggle Just-In-Time user provisioning for SSO-enabled organization.")
```

Comment:
Is this a one time operation, or can users disable it after enabling it? I think we can reword the flag description based depending on how this works.

Reply:
the latter. the description is consistent with the public api one

Reply:
```suggestion
	cmd.Flags().Bool("jit-enabled", false, "Toggle Just-In-Time (JIT) user provisioning for SSO-enabled organizations.")
```